packages/discovery-provider/src/utils/spl_audio.py

```diff
@@ -4,10 +4,16 @@
 SPL_TO_WEI = 10**10
 SPL_TO_WEI_PADDING = "0" * 10
 
+WEI_TO_INT = 10**8
```

Comment:
This is mildly confusing due to the comment above regarding sol/eth wei. The `SPL_TO_WEI` uses 10^10 but this constant (which reads as the other direction) uses 10^8?

Reply:
hm good call-out, can't remember why it's sometimes one and sometimes the other

Reply:
aha it's actually lamports for sol not wei!